### ISSUE TITLE
How to add a shortcut to start TexText?

### ISSUE THREAD

**Issue by tovrstra** [textext/textext#259]
**Is your feature request related to a problem? Please describe.**
It would be nice to have a shortcut to start TexText, especially when working on drawings with many equations.

**Describe the solution you'd like**
I tried making a file `~/.config/inkscape/default.xml` with the following contents:

```xml
<?xml version="1.0"?>
<keys name="Inkscape default">
    <bind key="l" modifiers="Primary,Alt" action="org.ekips.filter.textext" display="true" />
</keys>
```

This did not have the desired effect.

**Additional context**
The definition of custom shortcuts in inkscape is documented here: https://gitlab.com/inkscape/inkscape/-/blob/master/share/keys/inkscape.xml 

Maybe I missed something? Did anyone succeed in getting this to work? Thanks!

**Comment by jcwinkler** [comment]
Inkscape offers the Shortcut `ALT`+`Q` to launch the previously used extension. So you need only to open TexText once via the menu per session.

**Comment by tovrstra** [comment]
@jcwinkler Thanks for your suggestion. It helps a lot!

**Comment by sizmailov** [comment]
@tovrstra You can bind a custom shortcut in GUI. Go to `Edit` -> `Preferences` -> `Interface` -> `Keyboard Shortcuts`, find `Tex Text`in `Extensions` sections and add shortcut. Just tried with inkscape 0.92.

@jcwinkler Maybe it's worth adding to FAQ section?

**Comment by tovrstra** [comment]
@sizmailov Thanks for pointing this out. The usage of this GUI configuration of shortcuts in inkscape 1.0.1 is not entirely straightforward. I made it work by clicking the "Import" button in the shortcuts configuration. Then I selected an xml file with the contents shown in my top post. After that, the shortcut works as I initially wanted. 

Taking a closer look, inkscape stores the imported shortcuts in `~/.config/inkscape/keys/default.xml`, not `~/.config/inkscape/default.xml`, as I first thought. Manually copying the xml file there also works. Taking a second look at the comments in `/usr/share/keys/inkscape.xml`, this makes sense, but I did not get it from a first read. An FAQ item would indeed be helpful. In any case, thanks for all the help. I'm glad it works!

**Comment by sizmailov** [comment]
FYI: Keybindings GUI in inkscape 1.1-dev works exactly same as in 0.92. 

Not entirely sure why do you need to edit/move files manually, but happy to see you've managed it to work :)

**Comment by tovrstra** [comment]
Sorry, It was not entirely obvious how to use the GUI. I got it now: first one has to select the line of interest, then one has to click again on the line, within the shortcut column, then one can press the new key combination.

It is not very discoverable, because one has to guess that clicking in a blank spot in the table after selecting the line will have some effect. I tried just double clicking on the line before, but not in the right place, which had no effect.
